lost/src/main/java/com/mapzen/android/lost/internal/FusedLocationProviderApiImpl.java

```diff
@@ -123,6 +123,14 @@ public void shutdown() {
         shutdownAllEngines();
     }
 
+    public List<LocationListener> getListeners() {
```

Comment:
I think we still need this because as far as I can tell, `assertThat(shadowLocationManager.getRequestLocationUpdateListeners()).isEmpty();` will always be true when mock mode is set. Maybe we could set this visible for testing only?